springboard/utils.py

```diff
@@ -8,6 +8,16 @@
 from elasticgit.search import RepoHelper
 
 
+default_excluded_paths = ['/health/', '/api/notify/']
+
+
+def excluded_path(path, excluded_paths):
```

Comment:
Tiny nitpick on this being named `is_excluded_path()` since it's a test that returns a boolean.